templates/citizen_project/_sidebar.html.twig

```diff
@@ -64,14 +64,14 @@
     <div class="citizen_project__card">
         <h3 class="svg--black">{{ 'citizen_project.committees'|transchoice(citizen_project.approvedCommitteeSupports|length) }}</h3>
         <ul>
-            {% for committeeSupport in citizen_project.committeeSupports %}
-                {% if committeeSupport.isApprove %}
-                    <li class="text--body">
-                        {{ committeeSupport.committee }}
-                    </li>
-                {% endif %}
+            {% for committeeSupport in citizen_project.approvedCommitteeSupports %}
+                <li class="text--body">
+                    {{ committeeSupport.committee }}
+                </li>
             {% endfor %}
         </ul>
-        <a href="#">Notre comité souhaite aussi soutenir ce projet</a>
+        {% if app.user and is_granted('ROLE_SUPERVISOR') %}
```

Comment:
I think testing `if app.user` is implicit when using `is_granted`

Reply:
👍